util/mime-type.service.ts

```diff
@@ -0,0 +1,54 @@
+import { Injectable } from 'angular2/core';
+
+@Injectable()
+export class MimeTypeService {
```

Comment:
idk - we both added them at the same time, so they may just be the same thing, or close enough.
My thought had been that `app/shared` was app specific, where `util/` was stuff that was more generic and not app related. Thinking of it that way, env.ts probably should not be in there!